images/scripts/lib/build-deps.sh

```diff
@@ -23,7 +23,7 @@ case "$OS_VER" in
         ;;
 esac
 
-echo "$spec" | rpmspec -D "$OS_VER" -D 'version 0' -D 'cockpit_enable_python 1' --buildrequires --query /dev/stdin | sed 's/.*/"&"/' | tr '\n' ' '
+echo "$spec" | rpmspec -D "$OS_VER" -D 'version 0' -D 'enable_old_bridge 0' --buildrequires --query /dev/stdin | sed 's/.*/"&"/' | tr '\n' ' '
```

Comment:
This looks funny, but I've stared at this and the .spec very hard. This will only *add* the pip etc. build dependencies. So :+1:

Reply:
I put the output changes here:

https://github.com/cockpit-project/cockpit/pull/19330#issue-1896513267